PureBasicIDE/ProcedureBrowser.pb

```diff
@@ -202,6 +211,11 @@ Procedure UpdateProcedureList()
     
     ;StopGadgetFlickerFix(#GADGET_ProcedureBrowser)
     
+    ;- Added by mk-soft; Date 22.05.2020
```

Comment:
This comment is no longer necessary because Git stores this information about who added, changed, and removed which line.